docs/Apple_Silicon_Dev_Setup.md

```diff
@@ -0,0 +1,256 @@
+start menu will get you there quickly), and find `FMS.FielServer.Web`. Right click and hit "Start".
+
+![Starting the FMS web service](img/start_service.png)
+
+### Starting SQL Server after a Restart
+
+After you restart the VM, the SQL Server will not start back up on it's own. To start up the SQL Server
+instance manually, open up the "SQL Server Configuration Manager", select "SQL Server Services" in the menubar
+on the left. Then find "SQL Server (FRCSQLEXPRESS)", right click it and hit "Start".
+
+![Starting SQL Server](img/start_sql.png)
+
+### Selecting a USB Flash Drive for FMS
+
+Most versions of FMS offseason enforce the use of a USB Flash Drive as a backup location. Creating a flash drive
+for use in UTM is pretty difficult, though not impossible. However, the easiest solution is to cheese this by
+simply selecting a external drive/volume that's not readable (like the UTM tools installer), causing FMS to
+crash. Then, start it back up and you'll find your DB created! To the FMS developers: I'm very sorry for this.
+
+### Networking setup for a real-ish FMS network
+
+For testing and interfacing, ideally you want to be able to interact with the FMS at 10.0.100.5 just like a real
+field network. Thankfully, since we're running windows in a VM, that's pretty easy to do! Make sure your VM is
+fully shut down, and then click edit (or the settings icon) on your VM in UTM. Go to the "Network" tab in the
+edit dialog, and then set the settings to something like this:
+
+![UTM networking settings](img/utm_networking.png)
+
+The DHCP range os 10.0.100.4-10.0.100.100 is intentional - the first address in the DHCP pool is used as the
+gateway device, so your windows host will pick up the .5 address. The MAC address can be whatever - it does not
+to match the one in the screenshot.
+
+You will also need to alter the Windows Firewall settings from their defaults to be able to access the web
+server from the Mac host:
+
+![Disabled Windows firewall](img/windows_firewall.png)
+
+# 4. Install SSMS and create a Token
+
+To create a token for development, you will need the SQL Server Management Studio (SSMS), however just like SQL
+Server itself, the installer for this doesn't work on Arm64-based Windows hosts. Thankfully, somebody has
+created an installer that works on Arm64.
+
+## Install SSMS
+
+1. Paste in `https://github.com/snickler/Posh-SSMSInstaller/blob/main/Download-SSMS.ps1` in Edge, and click the
+   Download button in the top right.
+2. Open Powershell (NOT a regular command prompt) as administrator.
+3. Enable Powershell script execution with `Set-ExecutionPolicy unrestricted`
+4. `cd Downloads`
+5. Run `./Download-SSMS.ps1`
+6. When promped, type `R` and press enter to run the script. This will take a while to execute - be patient!
+7. If prompet to download a NuGet provder, hit "Y" and enter.
+8. When done, restart to complete the SSMS installation.
+
+## Create a FTA Token
```

Comment:
Would you mind pulling this into a separate doc, since it's generally applicable?